res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -376,6 +373,7 @@ Jockey3ME.effectSelectPush = function (channel, control, value, status, group) {
 	}
 }
 
+// This is control by Jogwheel Pitch bend
```

Comment:
The wording could be a bit clearer here. How about "This function handles the Pitch Bend mode of the jogwheel"